pkg/config/ebpf_tracer.go

```diff
@@ -114,70 +112,28 @@ type EBPFTracer struct {
 // Max: 8192 bytes.
 // Default: 0 (disabled).
 type EBPFBufferSizes struct {
-	HTTP     uint32 `yaml:"http" env:"OTEL_EBPF_BPF_BUFFER_SIZE_HTTP"`
-	MySQL    uint32 `yaml:"mysql" env:"OTEL_EBPF_BPF_BUFFER_SIZE_MYSQL"`
-	Postgres uint32 `yaml:"postgres" env:"OTEL_EBPF_BPF_BUFFER_SIZE_POSTGRES"`
+	HTTP     uint32 `yaml:"http" env:"OTEL_EBPF_BPF_BUFFER_SIZE_HTTP" validate:"lt=8192"`
```

Comment:
i think this only accepts powers of 2, right @mmat11 ?
in that case will it be correct to create a custom validator or something?